src/StackHCI/custom/stackhci.ps1

```diff
@@ -2438,15 +2470,6 @@ param(
         $registrationOutput = New-Object -TypeName PSObject
         $operationStatus = [OperationStatus]::Unused
         
-        try
```

Comment:
Just for confirm, is removing installation of NuGet expected?

Reply:
This script no longer installs any modules, so Nuget is not needed.